app/models/architecture.rb

```diff
@@ -4,6 +4,8 @@ class Architecture < ActiveRecord::Base
   friendly_id :name
   include Parameterizable::ByIdName
 
+  attr_accessible :name, :operatingsystem_ids
```

Comment:
We're using `https://github.com/rails/protected_attributes` , not strong_params here, in any case we want to put `strong_params` in later (right after this PR is merged).  https://github.com/theforeman/foreman/pull/2509 is tracking it, we should populate `strong_params` with the attributes from Apipie.